run.go

```diff
@@ -74,8 +74,13 @@ func (c *Client) GetRun(runID int) (Run, error) {
 // GetRuns returns the list of runs of projectID
 // validating the filters
 func (c *Client) GetRuns(projectID int, filters ...RequestFilterForRun) ([]Run, error) {
+	uri := "get_runs/" + strconv.Itoa(projectID)
+	if len(filters) > 0 {
+		uri = applyFiltersForRuns(uri, filters[0])
```

Comment:
we may want to loop through filters here and apply each or update the applyFiltersForRuns to accept a slice and do it there (applyFiltersForRuns seems to imply we're applying more than one filter)